config/config/wc-config.yaml

```diff
@@ -54,6 +54,7 @@ velero:
     - cert-manager
     - falco
     - fluentd
+    - fluentd-system
```

Comment:
We don't want to backup the the volume for audit logs?

Reply:
if it works to restore the volume with a new pod, yes makes sense to back it up

Reply:
Perhaps warrants a test?
Feel free to create an issue for it as I assume it might take some time to prepare and test.

Reply:
Issue created:
https://github.com/elastisys/compliantkubernetes-apps/issues/1567